examples/tutorials/advanced/websockets-example-MNIST-parallel/run_websocket_client.py

```diff
@@ -2,33 +2,35 @@
 import torch.nn as nn
 import torch.nn.functional as F
 from torchvision import transforms, datasets
+
 import logging
 import argparse
 import sys
 import asyncio
 import numpy as np
 
-if __name__ == "__main__":
```

Comment:
Marianne, there was a reason to have the call to basicConfig on the beginning of the file. It is ugly there, but it does not work if you put it later.
I didn't identify the source yet, but when doing import syft basicConfig seems to be set somewhere already. And any subsequent calls to basicConfig are ignored by the logging library.

Reply:
My bad, I imagined there was a good reason but I forgot to undo the change. Feel free to undo this.

Reply:
Also, @midokura-silvia can you add a comment to it, something like:

> There is a reason to have the call to basicConfig at the beginning of the file. It does not work if you put it later.